pages/learn/develop/dockerfile.md

```diff
@@ -101,6 +101,18 @@ Here are the supported machine names and architectures:
 
 {{> "general/deviceTypeNames"}}
 
+## Multiple Dockerfiles
+
+There are cases where just specifying a Dockerfile template doesn't work, as different systems or architectures may need different configuration and installation files. The solution to this problem is having multiple dockerfiles that are automatically selected based on the device it is going to be used.
```

Comment:
> specifying a Dockerfile template doesn't work

sounds a bit weird to be and I don't think gives off the correct message.

I'd say something along the lines of;

There are cases when you would need a higher granularity of control when specifying build instructions for different devices and architectures than a Dockerfile template can provide. An example of this would be when different configuration or installation files are required.